### DISCUSSION TITLE
Pull to refresh from bottom of issues / PRs

### DISCUSSION THREAD

**Discussion by mwarkentin** [community/community#5004] [Mobile] [answered]
Issues / PRs have pull to refresh at the top of the screen, however since comments and other items are laid out from oldest to newest, this means that you have to bounce all the way back to the top and then down again to refresh. 

It would be better if you could pull up at the bottom to refresh instead.

**Comment by eliperkins** [comment, marked as answer]
Thanks for this feedback @mwarkentin ! This is definitely something the team has been thinking about. I know it's frustrating to get the the bottom of a long issue or PR, only having to go up to the top again to pull-to-refresh.

I've jotted your notes down on an internal issue we have for this. Stay tuned for some updates!

**Reply by mwarkentin** [reply]
Thanks!

**Reply by francislavoie** [reply]
Just wanted to give this a bump and link to some frustration from this missing: https://twitter.com/_francislavoie/status/1584303903080087554

**Comment by OsingGeblek** [comment]
> Issues / PRs have pull to refresh at the top of the screen, however since comments and other items are laid out from oldest to newest, this means that you have to bounce all the way back to the top and then down again to refresh.
> 
> It would be better if you could pull up at the bottom to refresh instead.
